### DISCUSSION TITLE
How to specify the server address immediately when installing the program?

### DISCUSSION THREAD

**Discussion by jawakharlal** [rustdesk/rustdesk#9850] [Q&A] [answered]
I plan to use rustdesk only in a local network, so I would like to specify the server_id to the program when installing the MSI
Is it possible to do this?

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/discussions/763, go to [discord](https://discord.gg/nDceKgxnkV) please, please search there first, many guys have asked similar question because you are not the most special though you always think you are. Usually nobody will help you with build or self-hosting here.
